source/utilities.tex

```diff
@@ -15881,7 +15881,7 @@
   \tcode{is_constructible_v<T,}\br
   \tcode{Args...>} is \tcode{true} and the variable
   definition for \tcode{is_constructible}, as defined below, is known to call
-  no operation that is not trivial (~\ref{basic.types},~\ref{special}). &
+  no operation that is not trivial (\ref{basic.types},~\ref{special}). &
```

Comment:
We usually do `blah~(\ref` so it looks as though the `~` just got put in the wrong place here. Do we want to move it rather than removing it?